_data/docs.yml

```diff
@@ -90,6 +90,8 @@
       keywords : ["fridge monitoring","fridge"]
     - title: Leak Detector Installation Guide
       keywords : ["leak","water"]
+    - title: Environment Monitoring Installation
+      keywords : ["environment","environment monitoring","install"
```

Comment:
It might be interesting in this "specific installation" group to keep the installations in alphabetic order

Reply:
Good idea Julien

Reply:
@tidelbo